src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,55 @@
+package tasks;
+
+import exceptions.DataException;
+
+/**
+ * An abstract task. All tasks have a description and completion status.
+ * They start as incomplete, and can be marked as complete later.
+ */
+public abstract class Task {
+
+    // task description
+    private final String desc;
```

Comment:
I agree, since 'desc' is used in quite a few places, it may be better to use the full name 'description' so that it will be clearer in other parts of your code

Reply:
Fixed, thanks!